src/Symfony/Component/ObjectMapper/Transform/MapCollection.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Symfony\Component\ObjectMapper\Transform;
+
+use Symfony\Component\ObjectMapper\Exception\MappingException;
+use Symfony\Component\ObjectMapper\ObjectMapper;
+use Symfony\Component\ObjectMapper\ObjectMapperInterface;
+use Symfony\Component\ObjectMapper\TransformCallableInterface;
+
+/**
+ * @template T of object
+ *
+ * @implements TransformCallableInterface<object, T>
+ */
+class MapCollection implements TransformCallableInterface
+{
+    public function __construct(private ?ObjectMapperInterface $objectMapper = null)
```

Comment:
```suggestion
    public function __construct(private ?ObjectMapperInterface $objectMapper = new ObjectMapper())
```
Can we use this here?